.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Please **_rebase_** on master instead of **_merge_** and fix logical conflicts. 
```
Error:  Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project trino-delta-lake: Compilation failure: Compilation failure: 
Error:  /home/runner/work/trino/trino/plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePartitionsTable.java:[95,49] cannot find symbol
Error:    symbol:   method getSnapshot(io.trino.spi.connector.ConnectorSession,io.trino.spi.connector.SchemaTableName,java.lang.String,java.util.Optional<java.lang.Object>)
Error:    location: variable transactionLogAccess of type io.trino.plugin.deltalake.transactionlog.TransactionLogAccess
Error:  /home/runner/work/trino/trino/plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePartitionsTableProvider.java:[57,63] cannot find symbol
Error:    symbol:   method getSnapshot(io.trino.spi.connector.ConnectorSession,io.trino.spi.connector.SchemaTableName,java.lang.String,java.util.Optional<java.lang.Object>)
Error:    location: variable transactionLogAccess of type io.trino.plugin.deltalake.transactionlog.TransactionLogAccess
Error:  -> [Help 1]
```